docs/en_US/NNICTLDOC.md

```diff
@@ -650,3 +651,37 @@ Debug mode will disable version check function in Trialkeeper.
     ```bash
     nnictl --version
     ```
+
+<a name="hdfs"></a>
+
+![](https://placehold.it/15/1589F0/000000?text=+) `Manage hdfs`
+
+* __nnictl hdfs set__
+  * Description
+
+    set the host and userName of hdfs
+
+  * Usage
+
+    ```bash
+    nnictl hdfs set [OPTIONS]
```

Comment:
Since we only clean PAI job tmp dir on PAI HDFS, we should not use so general command name: 'nnictl hdfs clean',  suggest to rename it to ``` nnictl pai-tmp-dir clean```  or something else.

Reply:
As discussed in the meeting, we use `nnictl hdfs` command.